docs/extend/plugin_api.md

```diff
@@ -96,6 +96,43 @@ directory and activates it with a handshake. See Handshake API below.
 Plugins are *not* activated automatically at Docker daemon startup. Rather,
 they are activated only lazily, or on-demand, when they are needed.
 
+## Systemd socket activation
+
+Plugins may also be socket activated by `systemd`. The official [Plugins helpers](https://github.com/docker/go-plugins-helpers)
+natively supports socket activation. In order for a plugin to be socket activated it needs
+a `service` file and a `socket` file:
+
+```bash
+$ cat your-plugin.service
```

Comment:
small nit, but can we just put the contents of the unit file's in separate code blocks, and "your-plugin.service" outside of the code block?